BGAnimations/ScreenOptionsServiceChild overlay.lua

```diff
@@ -53,10 +57,11 @@ af[#af+1] = Def.ActorFrame{
 		InitCommand=function(self)
 			recommended_bmt = self
 
-			self:xy(- bg_width/2 + padding, bg_height/2 - padding)
-				:valign(1) -- bottom aligned
```

Comment:
this says bottom aligned on both valign(1) and valign(0), I suspect one of those is incorrect